### ISSUE TITLE
Selection of DECAY and MAX_ITER

### ISSUE THREAD

**Issue by WangBJ233** [YazhouZhu19/PAMI#4]
Hello author, in your github code train.sh DECAY=0.98, MAX_ITER=3000, But in your article "the initial learning rate is set to 1 × 10−3 with a step decay of 0.8 each 1000 iteration". I understand that DECAY=0.8 and MAX_ITER=1000 according to the statement in the paper. Which should be used for DECAY and MAX_ITER?

**Comment by hjklearn** [comment]
你好，我在复现这篇论文的时候，我也遇到了一些问题，结果复现不出来，可以相互学习一下吗
